gateway/src/main/java/org/georchestra/gateway/events/RabbitmqEventsAutoConfiguration.java

```diff
@@ -14,7 +14,7 @@
 @Configuration(proxyBeanMethods = false)
 @AutoConfigureAfter(GatewayAutoConfiguration.class)
 @ImportResource({ "classpath:rabbit-listener-context.xml", "classpath:rabbit-sender-context.xml" })
-@ConditionalOnExpression("${georchestra.gateway.security.enableRabbitmqEvents:true}")
+@ConditionalOnExpression("${enableRabbitmqEvents:true}")
```

Comment:
why not nesting the `enableRabbitmqEvents` flag under `georchestra.gateway.security` anymore ?

Reply:
replying to myself: because the flag goes into default.properties form the datadir: https://github.com/georchestra/datadir/pull/339/files#diff-7b382666b6f0b8f6c5e9fbc8b613e21e1c5c2a11639322caf55e4dc7024464e0R61 and would also be consumed console-side.